### PR TITLE
feat: add readme

### PR DESCRIPTION
**Pull request by sailorlqh**

### CODE REVIEW

**Review by yolocs — APPROVED** [review]
I suppose the example module link depends on the other PR?

**Comment by sailorlqh** [comment]
> I suppose the example module link depends on the other PR?

Yes, I will merge and rebase after the other PRs got merged.

**Review by sqin2019 — APPROVED** [review]

**Pull request merged**
